### PR TITLE
feature/refresh state intro text

### PR DESCRIPTION
**Pull request by oftheheadland**
## Related Issues:
* https://app.breeze.pm/projects/100762/cards/3163120

## Main Changes:
* Update the state intro text file with the most recent changes to the spreadsheet

## Steps To Test:
1. Navigate to state page and check the modified states
2. You might need to clear your cache to test this change. the file was cached locally for me

### CODE REVIEW

File: app/client/src/components/pages/State/lookups/introText.js [hunk 1]
```diff
@@ -697,24 +697,25 @@ export default {
...
...rtment of Energy and Environment, Division of Environmental Quality (DEQ), Office of Water Quality (OWQ) implements the Federal Clean Water Act through its water quality standards, surface water monitoring, 303(d), 305(b), TMDL, NPDES, 401 Certification, storm water permitting, compliance, and enforcement programs. The Water Quality Planning Branch (Planning) oversees water quality standards, monitoring, assessments, listing, reporting, and TMDL development programs. With the aid of the Compliance Branch, Planning monitors surface waters across the state with its ambient water quality network, ambient lake sampling program, and special projects. These monitoring data make up the bulk of Arkansas’s assessment database. However, the State routinely receives additional data from other entities including Universities, watershed groups, and state and federal agencies. Arkansas’s nonpoint, 319, program is implemented by DEQ’s sister agency Arkansas Natural Resources Commission (ANRC). Planning “special projects” include, but are not limited to, criteria development, use attainability analyses, mining reclamation, focused impairment issues, karst geology, and harmful algal blooms (HABS.',
```
Comment (cschwinderg) [inline app/client/src/components/pages/State/lookups/introText.js]: Just need to add a closing parentheses after HABS

```suggestion
      'The Arkansas Department of Energy and Environment, Division of Environmental Quality (DEQ), Office of Water Quality (OWQ) implements the Federal Clean Water Act through its water quality standards, surface water monitoring, 303(d), 305(b), TMDL, NPDES, 401 Certification, storm water permitting, compliance, and enforcement programs. The Water Quality Planning Branch (Planning) oversees water quality standards, monitoring, assessments, listing, reporting, and TMDL development programs. With the aid of the Compliance Branch, Planning monitors surface waters across the state with its ambient water quality network, ambient lake sampling program, and special projects. These monitoring data make up the bulk of Arkansas’s assessment database. However, the State routinely receives additional data from other entities including Universities, watershed groups, and state and federal agencies. Arkansas’s nonpoint, 319, program is implemented by DEQ’s sister agency Arkansas Natural Resources Commission (ANRC). Planning “special projects” include, but are not limited to, criteria development, use attainability analyses, mining reclamation, focused impairment issues, karst geology, and harmful algal blooms (HABS).',
```

Comment (oftheheadland) [reply app/client/src/components/pages/State/lookups/introText.js]: Thanks @cschwinderg! I'll fix that typo and double check the Wisconsin metrics with Kiki

Comment (oftheheadland) [reply app/client/src/components/pages/State/lookups/introText.js]: Kiki confirmed the Assessed Waters value should be 30% and the Arkansas typo is now corrected

Comment (cschwinderg) [reply app/client/src/components/pages/State/lookups/introText.js]: Thank you for making the change and checking up on that.

**Pull request merged**